api/src/main/java/io/strimzi/api/kafka/model/common/authentication/KafkaClientAuthenticationOAuth.java

```diff
@@ -166,6 +168,16 @@ public void setAccessToken(GenericSecretSource accessToken) {
         this.accessToken = accessToken;
     }
 
+    @Description("Path to the token file containing an access token to be used for authentication.")
```

Comment:
Is this required when using `type: serviceaccount-oauth`?

Reply:
It is an underlying capability which may be used with `oauth`. When using `serviceaccount-oauth` this value is automatically configured by the operator to the standard path of the serviceaccount token file.